modules/programs/git.nix

```
@@ -274,6 +300,11 @@ in {
   config = mkIf cfg.enable (mkMerge [
     {
       home.packages = [ cfg.package ];
+      assertions = [{
+        assertion = !(cfg.delta.enable && cfg.difftastic.enable);
+        message =
+          "Only one of 'git.delta.enable` or `git.delta.difftastic.enable` can be set to true at the same time.";
```

Oh, I really screwed that suggestion up. This isn't markdown.

```suggestion
          "Only one of 'git.delta.enable' or 'git.delta.difftastic.enable' can be set to true at the same time.";
```

---

Shouldn't the second one be 'git.difftastic.enable' instead of 'git.delta.difftastic.enable'?

---

Also misses "programs.". Perhaps something like

> Only one of 'programs.git.delta.enable' or 'programs.git.difftastic.enable' can be enabled at the same time.

---

I changed it for 'programs.git.delta.enable` or `programs.git.difftastic.enable`